libs/SalesforceHybrid/src/com/salesforce/androidsdk/phonegap/app/SalesforceHybridSDKManager.java

```diff
@@ -0,0 +1,160 @@
+		if (INSTANCE == null) {
+    		INSTANCE = new SalesforceHybridSDKManager(context, keyImpl, mainActivity, loginActivity, appType);
+    	}
+		initInternal(context);
+
+        // Upgrade to the latest version.
+        SalesforceHybridUpgradeManager.getInstance().upgrade();
+        EventsObservable.get().notifyEvent(EventType.AppCreateComplete);
+	}
+
+	/**
+	 * Initializes components required for this class
+	 * to properly function. This method should be called
+	 * by hybrid apps using the Salesforce Mobile SDK.
+	 *
+	 * @param context Application context.
+     * @param keyImpl Implementation of KeyInterface.
+	 */
+    public static void initHybrid(Context context, KeyInterface keyImpl) {
+		SalesforceHybridSDKManager.init(context, keyImpl, SalesforceDroidGapActivity.class,
+				LoginActivity.class, AppType.Hybrid);
+    }
+
+	/**
+	 * Initializes components required for this class
+	 * to properly function. This method should be called
+	 * by hybrid apps using the Salesforce Mobile SDK.
+	 *
+	 * @param context Application context.
+     * @param keyImpl Implementation of KeyInterface.
+     * @param loginActivity Login activity.
+	 */
+    public static void initHybrid(Context context, KeyInterface keyImpl,
+    		Class<? extends Activity> loginActivity) {
+		SalesforceHybridSDKManager.init(context, keyImpl, SalesforceDroidGapActivity.class,
+				loginActivity, AppType.Hybrid);
+    }
+
+	/**
+	 * Initializes components required for this class
+	 * to properly function. This method should be called
+	 * by hybrid apps that use a subclass of SalesforceDroidGapActivity.
+	 *
+	 * @param context Application context.
+     * @param keyImpl Implementation of KeyInterface.
+     * @param mainActivity Main activity.
+     * @param loginActivity Login activity.
+	 */
+    public static void initHybrid(Context context, KeyInterface keyImpl,
+    		Class<? extends SalesforceDroidGapActivity> mainActivity,
+    		Class<? extends Activity> loginActivity) {
+		SalesforceHybridSDKManager.init(context, keyImpl, mainActivity, loginActivity, AppType.Hybrid);
+    }
+    
+    /**
+     * Returns a singleton instance of this class.
+     *
+     * @return Singleton instance of SmartSyncSDKManager.
+     */
+    public static SmartSyncSDKManager getInstance() {
+    	if (INSTANCE != null) {
+    		return (SmartSyncSDKManager) INSTANCE;
+    	} else {
+            throw new RuntimeException("Applications need to call SmartSyncSDKManager.init() first.");
+    	}
+    }
+
+    @Override
+    protected void cleanUp(Activity frontActivity, Account account) {
+    	final UserAccount userAccount = SmartSyncUserAccountManager.getInstance().buildUserAccount(account);
+    	MetadataManager.reset(userAccount);
+
+    	/*
+    	 * We don't have to do a hard reset on the cache manager here, since
+    	 * the underlying database will be wiped in the super class.
+    	 */
+    	CacheManager.softReset(userAccount);
```

Comment:
You're right. I created SalesforceHybridSDKManager.java by copying SmartSyncSDKManager.java and then cleaning up - and I missed that method.